src/patternfly/components/InputGroup/input-group.scss

```diff
@@ -46,7 +46,6 @@
 
     &[aria-invalid="true"] {
       position: relative;
-      z-index: var(--pf-c-input-group--c-form-control--invalid--ZIndex);
```

Comment:
I'm guessing this has to do with the old border issue when the design was different. @mcoker might have some insight on this too. I'd say we can remove `position:relative;` here too if we aren't including a z-index.

Reply:
yep it was  from when the error input had a red border all around, and if you had, say, a dropdown + invalid input in an input group, the right border from the dropdown hid the red left border on the input. Agreed, don't think we need `position: relative;` here. I also don't think we need `position: relative` on `.pf-c-form-control`. Assuming we can remove that, we can also remove `position: static` from `.pf-c-input-group .pf-c-form-control`

Reply:
Ah when I remove `position: static` from  `.pf-c-input-group .pf-c-form-control` we get this: 

<img width="820" alt="Screen Shot 2019-06-07 at 1 47 25 PM" src="https://user-images.githubusercontent.com/20118816/59123285-e4193600-892a-11e9-8633-3196acb0d6a5.png">

Reply:
so going to keep that in :)

Reply:
Did you also remove `position: relative` from `.pf-c-form-control`?

Reply:
yes @mcoker

Reply:
fixed thanks, didnt realize you meant from that component